src/pybind/mgr/dashboard_v2/tests/test_auth.py

```diff
@@ -6,12 +6,11 @@
 import time
 from unittest import skipIf
 
-from mock import patch
-
-from .helper import ControllerTestCase
+from .helper import ControllerTestCase, skip_if_loaded_by_mgr
 from ..tools import Session
 
 
+@skip_if_loaded_by_mgr
```

Comment:
Regardless of whether this whole "skip if loaded by the mgr" actually has some kernel of sense to it, adding these decorators is a different logical change from the rest of the patch, and should be a different commit.